lib/internal/http2/core.js

```diff
@@ -960,6 +961,35 @@ const validateSettings = hideStackFrames((settings) => {
   }
 });
 
+// Wrap a typed array in a proxy, and allow selectively copying the entries
+// that have explicitly been set to another typed array.
+function trackAssignmentsTypedArray(typedArray) {
+  const modifiedEntries = new Uint8Array(typedArray.length);
```

Comment:
It's probably worth cache the length to avoid calling the getter more than once.
```suggestion
  const typedArrayLength = TypedArrayPrototypeGetLength(typedArray);
  const modifiedEntries = new Uint8Array(typedArrayLength);
```